lms/static/scripts/frontend_apps/icons.js

```diff
@@ -13,4 +13,5 @@ export default {
   // Shared icons
   'arrow-left': require('@hypothesis/frontend-shared/images/icons/arrow-left.svg'),
   'arrow-right': require('@hypothesis/frontend-shared/images/icons/arrow-right.svg'),
+  cancel: require('@hypothesis/frontend-shared/images/icons/cancel.svg'),
```

Comment:
It's on my long list to consolidate how shared icons are registered and managed; I don't think this is the way we want to do things in the long run.